backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Integration/Clients/ResourceRegistryClient.cs

```diff
@@ -161,16 +159,15 @@ public async Task<List<ServiceResource>> GetResourceList()
         }
 
         /// <summary>
-        ///     Get resources of a given type
+        ///     Gets all MaskinportenSchemas
         /// </summary>
-        /// <param name="resourceType"> the resource type</param>
-        /// <returns></returns>
-        public async Task<List<ServiceResource>> GetResources(ResourceType resourceType)
+        /// <returns>MaskinportenSchemas</returns>
+        public async Task<List<ServiceResource>> GetMaskinportenSchemas()
         {
             List<ServiceResource> resources = new List<ServiceResource>();
-            ResourceSearch resourceSearch = new ResourceSearch();
-            resourceSearch.ResourceType = resourceType;
-            string endpointUrl = $"search?ResourceType={(int)resourceType}";
+            
+            // Weird enough it's not possible to filter on AltinnApp or GenericAccessResource for this endpoint
+            string endpointUrl = $"search?ResourceType={(int)ResourceType.MaskinportenSchema}";
```

Comment:
I tried running this locally with the actual AT23 Resource Registry again and I still got `Resource Not Found` on this call. After looking at it for a bit I realized that it's because part of the path is gone here.
Add resource to the path and it should work as intended 😊
```suggestion
            string endpointUrl = $"resource/search?ResourceType={(int)ResourceType.MaskinportenSchema}";
```

Reply:
Woops, thanks for looking into it 🥹